core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -429,6 +431,12 @@ export class NavigationComponent {
       });
   }
 
+  onClickGetAllVersions() {
+    this.workflowVersionService.retrieveVersionsOfWorkflow(<number>this.workflowActionService.
```

Comment:
`this.workflowActionService.getWorkflowMetadata()?.wid` is `number | undefiend`, and this code is force-casting it to `number`, this will cause bugs when the `wid` is not set, the code should handle the case where it's undefined